src/components/message-box/index.tsx

```diff
@@ -59,7 +59,7 @@ const MessageBox: React.FC<MessageBoxProps> = ({
       {retryFunction && (
         <ThemeText
           style={styles.retryText}
-          type="body__link"
+          type="body__primary--underline"
```

Comment:
Shouldn't try again text have same color as main body text? Contrast issues?
![image](https://user-images.githubusercontent.com/4932625/112198638-9d82f180-8c0d-11eb-80d6-6cf1f1aea7ac.png)

Reply:
Saw this also. Think it's an issue from a previous PR (@gorandalum ) but can fix it as part of this aswell.

Reply:
Should be solved now